sirepo/job_supervisor.py

```diff
@@ -299,13 +299,18 @@ def _purge_sim(db_file):
             if d.lastUpdateTime > _too_old:
                 return
             if d.status == job.JOB_RUN_PURGED:
+                cls._purged_jids_cache.add(db_file.purebasename)
                 return
             p = sirepo.simulation_db.simulation_run_dir(d)
             pkio.unchecked_remove(p)
-            d.status = job.JOB_RUN_PURGED
-            cls.__db_write_file(d)
+            _update_db(d)
```

Comment:
Why a function here? It seems to only be used once.

Reply:
I just liked the abstraction, not that it hid much... removed

https://github.com/radiasoft/sirepo/pull/2521/commits/bf9c680f975b12ee38bfa0b0e494fb3e3abe90f7